### PR TITLE
Add KorQPair dataset

### PR DESCRIPTION
**Pull request by jaketae**
This PR adds a [Korean paired question dataset](https://github.com/songys/Question_pair) containing labels indicating whether two questions in a given pair are semantically identical. This dataset was used to evaluate the performance of [KoGPT2](https://github.com/SKT-AI/KoGPT2#subtask-evaluations) on a phrase detection downstream task.

### CODE REVIEW

**Comment by cceyda** [comment]
dummy data is missing

**Comment by jaketae** [comment]
Hey @cceyda, thanks for pointing that out. I thought I'd added it, but seems like that wasn't the case. Just pushed a new commit with the dummy data.

File: datasets/kor_qpair/kor_qpair.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+_HOMEPAGE = "https://github.com/songys/Question_pair"
+
+_LICENSE = "The MIT License (MIT)"
+
+_URL = "https://raw.githubusercontent.com/songys/Question_pair/master/"
+_URLs = {key: f"{_URL}{key}.txt" for key in ("train", "test", "validation")}
+
+
+class KorQPair(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/kor_qpair/kor_qpair.py:45]:
```suggestion
class KorQpair(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `kor_qpair`

File: datasets/kor_qpair/dataset_infos.json [hunk 2]
```diff
@@ -0,0 +1 @@
... "ClassLabel"}}, "post_processed": null, "supervised_keys": null, "builder_name": "kor_q_pair", "config_name": "default", "version": {"version_str": "1.1.0", "description": null, "major": 1, "minor": 1, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 515365, "num_examples": 6136, "dataset_name": "kor_q_pair"}, "test": {"name": "test", "num_bytes": 63466, "num_examples": 758, "dataset_name": "kor_q_pair"}, "validation": {"name": "validation", "num_bytes": 57242, "num_examples": 682, "dataset_name": "kor_q_pair"}}, "download_checksums": {"https://raw.githubusercontent.com/songys/Question_pair/master/train.txt": {"num_bytes": 441758, "checksum": "addf4c88b19066ad55a09f466c5c3f6428b3f2bd5d55100d3ca5b93d9d8daff9"}, "https://raw.githubusercontent.com/songys/Question_pair/master/test.txt": {"num_bytes": 54395, "checksum": "0d5738107ea3742de4dcaf3a358b6bfa4818781bc6aec11a9956a3fbe35f8b61"}, "https://raw.githubusercontent.com/songys/Question_pair/master/validation.txt": {"num_bytes": 49083, "checksum": "3928c783f561f36547fd8e0c9d78a6e1d6013db3debcf559c4135aa5536badea"}}, "download_size": 545236, "post_processing_size": null, "dataset_size": 636073, "size_in_bytes": 1181309}}
```
Comment (lhoestq) [inline datasets/kor_qpair/dataset_infos.json:1]:
```suggestion
{"default": {"description": "This is a Korean paired question dateset containing labels that denote whether two questions in a given pair are semantically identical.\n", "citation": "@misc{Song:2018,\n  title     = \"Paired Question v.2\",\n  authors   = \"Youngsook Song\",\n  publisher = \"GitHub\",\n  year      = \"2018\"\n}\n", "homepage": "https://github.com/songys/Question_pair", "license": "The MIT License (MIT)", "features": {"question1": {"dtype": "string", "id": null, "_type": "Value"}, "question2": {"dtype": "string", "id": null, "_type": "Value"}, "is_duplicate": {"num_classes": 2, "names": ["0", "1"], "names_file": null, "id": null, "_type": "ClassLabel"}}, "post_processed": null, "supervised_keys": null, "builder_name": "kor_qpair", "config_name": "default", "version": {"version_str": "1.1.0", "description": null, "major": 1, "minor": 1, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 515365, "num_examples": 6136, "dataset_name": "kor_qpair"}, "test": {"name": "test", "num_bytes": 63466, "num_examples": 758, "dataset_name": "kor_qpair"}, "validation": {"name": "validation", "num_bytes": 57242, "num_examples": 682, "dataset_name": "kor_qpair"}}, "download_checksums": {"https://raw.githubusercontent.com/songys/Question_pair/master/train.txt": {"num_bytes": 441758, "checksum": "addf4c88b19066ad55a09f466c5c3f6428b3f2bd5d55100d3ca5b93d9d8daff9"}, "https://raw.githubusercontent.com/songys/Question_pair/master/test.txt": {"num_bytes": 54395, "checksum": "0d5738107ea3742de4dcaf3a358b6bfa4818781bc6aec11a9956a3fbe35f8b61"}, "https://raw.githubusercontent.com/songys/Question_pair/master/validation.txt": {"num_bytes": 49083, "checksum": "3928c783f561f36547fd8e0c9d78a6e1d6013db3debcf559c4135aa5536badea"}}, "download_size": 545236, "post_processing_size": null, "dataset_size": 636073, "size_in_bytes": 1181309}}
```

`kor_q_pair` -> `kor_qpair`

**Review by lhoestq — APPROVED** [review]
Awesome thank you :)

**Pull request merged**